Gemfile

```diff
@@ -4,3 +4,7 @@ gemspec
 
 # This here to make sure appraisal works with Rails 3.0.0.
 gem "thor", "~> 0.14.0"
+
+# Development gems
+gem "activesupport", ">= 3.2.21"
+gem "rspec", "~> 3.0"
```

Comment:
Would a group be better used here?

```suggestion
group :development, :test do
  gem "activesupport", ">= 3.2.21"
  gem "rspec", "~> 3.0"
end
```

Reply:
I have no opinions on this. I can scope them if you prefer.

Reply:
Done :)